src/lib_ccx/params.c

```diff
@@ -1698,6 +1724,19 @@ int parse_parameters(struct ccx_s_options *opt, int argc, char *argv[])
 			set_output_format(opt, argv[i]);
 			continue;
 		}
+		if (strncmp(argv[i], "--out", 5) == 0)
```

Comment:
Why this new block? It's identical to -o so you could just add the variants to line 1702.

Reply:
This is for output format (previously -out-format) so not sure what you mean by putting it with -o/--outputfilename